go.mod

```diff
@@ -39,6 +38,7 @@ require (
 )
 
 require (
+	github.com/algorand/avm-abi v0.2.0 // indirect
```

Comment:
Not sure.  I added this via `go get` but updated the mod file manually to be a direct require.

Reply:
I believe you can run `go mod tidy` to have go update the dependencies to match their usage in the codebase. I'd recommend still running it locally, since it might reorder the dependencies